package.json

```diff
@@ -105,14 +105,14 @@
     "@types/marked": "0.6.5",
     "@types/moment-timezone": "0.5.12",
     "@types/node": "12.12.5",
-    "@types/react": "16.9.19",
+    "@types/react": "^16.9.23",
     "@types/react-bootstrap": "0.32.20",
     "@types/react-color": "3.0.1",
     "@types/react-dom": "16.9.5",
     "@types/react-redux": "7.1.4",
     "@types/react-router-dom": "5.1.2",
     "@types/react-select": "3.0.10",
-    "@types/react-transition-group": "4.2.3",
+    "@types/react-transition-group": "^4.2.4",
```

Comment:
I think the consensus is - use exact versions and not "^". Right?

Reply:
Yes good point - I'm separately going to propose changing that consensus, but we shouldn't do that in this PR.